include/picongpu/particles/ionization/byField/IonizationCurrent/IonizationCurrent.hpp

```diff
@@ -0,0 +1,76 @@
+/* Copyright 2020 Jakob Trojok
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/simulation_defines.hpp"
+#include "picongpu/particles/ParticlesFunctors.hpp"
+#include "picongpu/fields/FieldE.hpp"
+#include "picongpu/particles/ionization/byField/IonizationCurrent/IonizerReturn.hpp"
+#include "picongpu/particles/ionization/byField/IonizationCurrent/JIonizationCalc.hpp"
+#include "picongpu/particles/ionization/byField/IonizationCurrent/JIonizationAssignment.hpp"
+
+
+
+namespace picongpu
+{
+namespace particles
+{
+namespace ionization
+{
+    template<
+        typename T_Acc,
+        typename T_DestSpecies,
+        unsigned T_Dim
+    >
+    struct IonizationCurrent<T_Acc, T_DestSpecies, T_Dim, ::picongpu::ionizationCurrent::EnergyConservation>
+    {
+        using ValueType_E = FieldE::ValueType;
+
+        template<typename T_JBox>
+
+        HDINLINE void
+        operator()(IonizerReturn retValue, float_X const weighting, T_JBox jBoxPar, ValueType_E eField, T_Acc const & acc, floatD_X const pos)
+        {
+            auto ionizationEnergy = weighting * retValue.ionizationEnergy * ::picongpu::SI::ATOMIC_UNIT_ENERGY / ::picongpu::UNIT_ENERGY; // Umrechnen in PIConGPU
+            /* calculate ionization current at particle position */
```

Comment:
correct: /* calculate ionization current density at particle position */ 
**AND**
divide `jIonizationPar` by `::picongpu::CELL_VOLUME` (or do it in `JIonizationCalc::operator()`)